src/lib.rs

```diff
@@ -1450,6 +1450,20 @@ impl<W: io::Write> Ansi<W> {
     }
 }
 
+impl WriteColor for io::Sink {
+    fn supports_color(&self) -> bool {
+        true
```

Comment:
I think this should probably be `false`? Why did you return `true` here?

Reply:
I was unsure if there is ever a situation where you absolutely need a sink which supports color. I can change it to `false`.

Reply:
What would a sink that supports color look like? It never writes anything so it would seem to me that it can't support color?

Reply:
Hmm, sorry I might have mistaken what `support_color` meant. I thought `support_color` meant you can write colored output to the sink without encountering any problems.